Source/WebCore/Modules/ShapeDetection/DetectedBarcode.idl

```diff
@@ -26,7 +26,8 @@
 // https://wicg.github.io/shape-detection-api/#dictdef-detectedbarcode
 
 [
-    EnabledBySetting=ShapeDetection
+    EnabledBySetting=ShapeDetection,
+    JSGenerateToJSObject
```

Comment:
because this object is returned from C++ to javascript (as opposed to just being delivered from javascript to C++), so it needs to have the toJSObject() function generated for it. This is the flag that tells our generators to do so.